man/na.omit.data.table.Rd

```diff
@@ -1,41 +1,59 @@
 \name{na.omit.data.table}
 \alias{na.omit.data.table}
 \alias{na.omit}
-\title{ Remove rows with missing values on columns specified }
+\alias{is_na}
+\alias{any_na}
+\title{Find missing values per row and remove rows with missing values on columns specified}
 \description{
-    This is a \code{data.table} method for the S3 generic \code{stats::na.omit}. The internals are written in C for speed. See examples for benchmark timings.
-
-    \code{bit64::integer64} type is also supported.
+  \code{is_na()} checks a \code{data.table} for \code{NA} values in row-wise fashion, returning \code{TRUE} if a given row contains a \code{NA} entry and \code{FALSE} otherwise. This corresponds to \code{rowSums(is.na(.))}, but implemented in \code{C} for speed. \code{any_na()} returns the equivalent of \code{any(is_na(.))}, but again is implemented in \code{C} for speed. In order to remove rows containing \code{NA} entries, an S3 generic function to \code{stats::na.omit} if provided for \code{data.table} objects. This internally calls \code{is_na()}. See examples for benchmark timings. \code{bit64::integer64} type is also supported.
```

Comment:
```suggestion
  \code{is_na(DT)} checks the \code{data.table} for \code{NA} values in row-wise fashion, returning a logical vector of length `nrow(DT)`, with each element being \code{TRUE} for rows contains an \code{NA} entry and \code{FALSE} otherwise. This is equivalent to \code{as.logical(rowSums(is.na(DT)))}, but implemented in \code{C} for speed. \code{any_na(DT)} returns the equivalent of \code{any(is_na(DT))}, but again is implemented in \code{C} for speed. In order to remove rows containing \code{NA} entries, an S3 generic function to \code{stats::na.omit} if provided for \code{data.table} objects. This internally calls \code{is_na()}. See examples for benchmark timings. \code{bit64::integer64} type is also supported.
```

Reply:
I don't have permission to accept this suggestion on this fork. Seem like a GitHub GUI permission issue, because I can commit to the fork using command line. So I'll commit it separately. Thanks.
![image](https://user-images.githubusercontent.com/2779998/59892876-c1a60480-938f-11e9-87d2-c805846979cf.png)